test/integration/topic_browsing_test.rb

```diff
@@ -61,7 +61,7 @@ def oil_and_gas_topic_item(params = {})
     )
 
     visit "/topic/oil-and-gas"
-    assert page.has_title?("Oil and gas - GOV.UK")
+    assert page.has_title?("Oil and gas: detailed information - GOV.UK")
```

Comment:
Should we add an equivalent test to check subtopic pages title, since that has changed as well?